include/chibi/sexp.h

```diff
@@ -683,7 +690,7 @@ SEXP_API sexp sexp_make_flonum(sexp ctx, float f);
 #define sexp_flonump(x)      (sexp_check_tag(x, SEXP_FLONUM))
 #define sexp_flonum_value(f) ((f)->value.flonum)
 #define sexp_flonum_bits(f) ((f)->value.flonum_bits)
-sexp sexp_make_flonum(sexp ctx, double f);
```

Comment:
While `SEXP_API` might not apply in other instances, wouldn't it be needed here? If not, I'll simply withdraw the pull request and look for another way to let the linker know about those functions (Windows DLLs need these `extern` declarations).